examples/connext_dds/build_systems/cmake/CMakeLists.txt

```diff
@@ -14,15 +14,6 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
 )
 
-# Find the ConnextDDS libraries. This will look fo the core and API libraries
```

Comment:
why do we need to delete this? I think that in `develop` we should require Connext DDS 6.1.0